integration/popular_docker_hub_images/popular_docker_images.bats

```diff
@@ -21,475 +21,635 @@ setup() {
 	clean_env
```

Comment:
@egernst this function is using docker
https://github.com/kata-containers/tests/blob/2.0-dev/lib/common.bash#L221

Reply:
addressed.